paywall/src/__tests__/data-iframe/cache/localStorage-no-localStorage.test.js

```diff
@@ -14,6 +24,16 @@ describe('localStorage cache', () => {
       }
     })
 
+    it('getReadOnly', async () => {
+      expect.assertions(1)
+
+      try {
+        await getReadOnly()
+      } catch (e) {
+        expect(e.message).toBe('Cannot get value from localStorage')
```

Comment:
Depending on the types of errors possible it might be worth while to move the  error message in the test to its own variable to be referenced in the subsequent tests